README.md

```diff
@@ -1420,3 +1420,6 @@ My name is [Nastya](https://github.com/AsyaYeromina) Happy to be the part of suc
 **YOU ARE SO COOL, KOTTANS!!!**
 ![cat-box](https://i.pinimg.com/564x/b5/1e/1b/b51e1b639edc459ac49b574ed69367ae.jpg)
 
+## Hello ALL! 
```

Comment:
Hi! Wonderful job!
But please setup your code editor to add empty line at the end of the file, it helps next contributors

Reply:
I don't understand why because i already add empty line at the end of the file?

Reply:
When your code editor or IDE add blank line automatically there is no need to check it by yourself every PR.
GH don't show empty line in the file, but OK I'll check locally

Reply:
Checked locally, everything looks fine for me, here is a [thread ](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)that explains why do we need a blank line at the end of the file.